core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -266,9 +268,54 @@ class ConsumerSettings[K, V] @InternalApi private[kafka] (
     val enrichAsync: Option[ConsumerSettings[K, V] => Future[ConsumerSettings[K, V]]],
     val consumerFactory: ConsumerSettings[K, V] => Consumer[K, V],
     val connectionCheckerSettings: ConnectionCheckerSettings,
-    val partitionHandlerWarning: FiniteDuration
+    val partitionHandlerWarning: FiniteDuration,
+    val resetProtectionSettings: ResetProtectionSettings
 ) {
 
+  def this(
```

Comment:
It breaks the CI otherwise with
```
[error] akka-stream-kafka: Failed binary compatibility check against com.typesafe.akka:akka-stream-kafka_2.13:2.1.0-M1! Found 1 potential problems (filtered 24)
[error]  * method this(scala.collection.immutable.Map,scala.Option,scala.Option,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.Duration,java.lang.String,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.concurrent.duration.FiniteDuration,scala.Option,scala.Function1,akka.kafka.ConnectionCheckerSettings,scala.concurrent.duration.FiniteDuration)Unit in class akka.kafka.ConsumerSettings does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("akka.kafka.ConsumerSettings.this")
[error] java.lang.RuntimeException: Failed binary compatibility check against com.typesafe.akka:akka-stream-kafka_2.13:2.1.0-M1! Found 1 potential problems (filtered 24)
[error] 	at scala.sys.package$.error(package.scala:30)
```

Reply:
Ah, that's ok. Add a MiMa exception to squash the error.

Reply:
Just an issue with MiMa.

Reply:
Bump. Let's drop `this` overload and add the MiMa exception.